### DISCUSSION TITLE
Unable to expose service - I'm using helm chart for kubernetes deployment

### DISCUSSION THREAD

**Discussion by karthiknagraj** [jertel/elastalert2#382] [Q&A] [answered]
Hi, 
I'm using elastalert for quite some time now (https://github.com/bitsensor/elastalert). Since the support was stopped, I want to migrate to using this project. But unfortunately, I'm unable to expose the service in Kubernetes deployment. In my existing helm template files, I replaced the image with your docker hub image  **jertel/elastalert2:2**. Can you please help ?

**Comment by nsano-rururu** [comment, marked as answer]
@karthiknagraj 

By the way, there is johnsusek/elastalert-server of bitsensor/elastalert fork. 
I have the latest elastalert 2.1.2 installed.
https://hub.docker.com/r/praecoapp/elastalert-server
https://github.com/johnsusek/elastalert-server/blob/master/DockerImageLog.md

**Reply by nsano-rururu** [reply]
ElastAlert 2 Helm Chart for Kubernetes
https://github.com/jertel/elastalert2/blob/master/chart/elastalert2/README.md

**Reply by nsano-rururu** [reply]
jertel / elastalert 2: 2 has not been released. The latest is 2.1.2.

**Reply by jertel** [reply]
@nsano-rururu `jertel/elastalert2:2` is a valid Docker tag. It represents the latest in the 2.x line. Documented here: https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-docker-container

@karthiknagraj Please provide more details about the problem, including configuration, errors and logs. Also, as @nsano-rururu mentioned, you might need to pursue the praeco fork if you're interested in using the Bitsensor version with built-in API.

**Reply by karthiknagraj** [reply]
Thank you @nsano-rururu, @jertel praeco fork works for me. While testing it out, I came across one issue but https://github.com/johnsusek/elastalert-server page doesn't have an issues section to raise an issue.

**Reply by nsano-rururu** [reply]
@karthiknagraj 

We apologize for any inconvenience.
Please write the issue of [johnsusek/praeco](https://github.com/johnsusek/praeco) issue. I will respond.
That's why I respond
Maintenance of johnsusek/praeco and johnsusek/elastalert-server has been taken over from johnsusek since August last year. It is treated as a joint maintainer.
Is it the policy of johnsusek, the issue is valid only for praeco. I have permission to close issues, capture pull requests, and push to master, but I don't have all permissions so I can't change it.

**Reply by nsano-rururu** [reply]
@karthiknagraj 

You can check the settings I use to check the operation of praeco and elastalert-server by visiting the site below.
https://github.com/johnsusek/praeco/wiki/docker-compose-sample(telegram)
